src/context/playListContext.jsx

```diff
@@ -0,0 +1,125 @@
+import { createContext, useContext, useState } from 'react'
+import axios from 'axios'
+import { useAuth } from 'context'
+
+const playlistContext = createContext({})
+
+function PlayListProvider ({ children }){
+
+    const [playlists, setPlayLists] = useState([])
+    const { authState: {token} } = useAuth()
+
+    function checkIfPresent (name, state){
+        return state.some(item => item === name)
+    }
+
+    const addPlaylist = async( item ) => {
+        if(item==''){
+            alert('Playlist name cannot be empty. Please give a valid name.')
+        }else{
+        if(!checkIfPresent(item, playlists.map(item => item.title))){
+        try{
+            const response = await axios.post("/api/user/playlists",
+            {
+                playlist: {title: item , description:"bar bar bar" }
+                },
+            {
+                headers: {
+                    authorization: token,   
+                },
+            }
+            )
+            setPlayLists(response.data.playlists);
+        }
+        catch(error){
+            console.log(error)

```

Comment:
remove consol log

Reply:
I have not created toast component yet for showing error messages. That is why I've used console.log in places of error. Will make sure to remove them once toast is ready.